ecs-agent/tmds/handlers/fault/v1/handlers/handlers_test.go

```diff
@@ -110,15 +110,19 @@ var (
 		"DelayMilliseconds":  delayMilliseconds,
 		"JitterMilliseconds": jitterMilliseconds,
 		"Sources":            ipSources,
+		"SourcesToFilter":    ipSourcesToFilter,
 	}
 
 	happyNetworkPacketLossReqBody = map[string]interface{}{
-		"LossPercent": lossPercent,
-		"Sources":     ipSources,
+		"LossPercent":     lossPercent,
```

Comment:
Can we have one more happy test case where no `SourcesToFilter` is provided?

Reply:
This is covered in the "unknown-request-body-no-existing-fault-no-allowlist-filter" case. We pass in an empty array as SourcesToFilter

Reply:
I see. Thanks!